tests/test_rtspstarttimerealtime.py

```diff
@@ -15,4 +15,17 @@ def test_rtsp_start_time_realtime(self):
             'stimeout': '5000000'  # socket timeout to 5s (default is unlimited)
         }
         with av.open(RTSP_STREAM, mode="r", options=stream_options) as container:
+            # wait for proper value set by RTSP in libavformat
+            while container.demux(container.streams.video[0]):
+                # it is okay to have None type at first dozens packets
+                if container.start_time_realtime is None:
+                    continue
+                # also sometimes it could be "undefined"
+                elif container.start_time_realtime <= 0:
```

Comment:
Why would it be strictly less than zero? I'd be really careful with conditional tests, the only "if" I would include is the one that skips None timestamps.

Reply:
internally (libavformat/rtsp.c) `start_time_realtime` is set to `AV_NOPTS_VALUE`  ` ((int64_t)UINT64_C(0x8000000000000000))`

Reply:
When i worked with ffmpeg directly (C code), negatives or incorrect values happened.
In PyAV are nice `None` values :)

Reply:
And i removed a second "if" in last commit. _All checks have passed_